extensions/getsound/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "getsound",
+  "title": "GetSound",
+  "description": "Get focused with GetSound.AI",
+  "icon": "logo.png",
+  "author": "asgudev",
+  "access": "public",
+  "owner": "getsound-ai",
+  "categories": ["Productivity",  "Media", "Fun"],
+  "contributors": [
+    "adigold1",
+    "lior_nitzan"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "play",
+      "title": "Focus",
+      "subtitle": "Start Music",
```

Comment:
The `title` should represent the main action that this command offers. In this case, it is to "Start Music". And the `subtitle` can be used to provide more information about the command, I think in this case it can be left empty, and it will use the extension name instead. 

```json
"title": "Start Music"
```

![(ಠ ›ಠ) — @peduarte — 1kaSWyip002426@2x](https://github.com/raycast/extensions/assets/372831/2528d08f-d7e5-4a37-b834-a79a0649c6a6)